src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -25,6 +25,7 @@
 public class DefaultRunConfiguration implements RunConfiguration {
 
   private List<File> appYamls;
+  private List<File> serviceDirectories;
```

Comment:
can we just call this services, especially if we're going to be at some point in the future dev_appserver2